__tests__/Place/place.test.ts

```diff
@@ -0,0 +1,122 @@
+import { Place } from "../../src/db/types";
+import { dbConnection } from "../../src/db/mongoConnection";
+import { getPlaceByID, addPlace } from "../../src/db/Place/place";
+import { MongoServerError } from "mongodb";
+
+beforeAll(async () => {
+	const { _db, _connection } = await dbConnection();
+	await _db.dropDatabase();
+	await _connection.close();
+
+	try {
+		await addPlace({
+			_id: "faketestid1",
+			avgBraille: null,
+			avgFontReadability: null,
+			avgGuideDogFriendly: null,
+			avgNavigability: null,
+			avgStaffHelpfulness: null,
+		});
+	} catch (e) {
+		if (e instanceof MongoServerError) {
+			console.log("MONGOSERVERERROR: Something went wrong while trying to connect to Mongo");
```

Comment:
I don't think a log message is a strong enough response to a broken test - we should probably fail the test with this message instead

Reply:
How do you force a jest test to fail? Wouldn't it just fail anyway when it gets to the assertion? Isn't that the point?

Reply:
Yeah I guess it would, okay we can leave it like this